### PR TITLE
Upload Multiple Entity Read Files to specified big query dataset.

### PR DESCRIPTION
**Pull request by michalslowikowski00**
---
Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

**Comment by potiuk** [comment]
Still some checks are failing :(

File: airflow/providers/google/marketing_platform/example_dags/example_display_video.py [hunk 1]
```diff
@@ -99,9 +104,18 @@
...
+        task_id="upload_line_items", bucket_name=BUCKET, object_name=OBJECT_NAME,
     )
-    # [END howto_google_display_video_upload_line_items_operator]
+    # [END howto_google_display_video_upload_multiple_entity_read_files_to_big_query]
+
+    # [START howto_google_display_video_upload_multiple_entity_read_files]
+    upload_erf_to_bq = GCSToBigQueryOperator(
+        task_id='upload_erf_to_bq',
+        bucket=BUCKET,
+        source_objects=[f"gdbm-{GMP_PARTNER_ID}/entity/{{{{ ds_nodash }}}}.*.{ENTITY_TYPE}.json"],
```
Comment (turbaszek) [inline airflow/providers/google/marketing_platform/example_dags/example_display_video.py:115]: Can we create a helper factory function that will help users easily generate those stings?
```python
def erf_bucket(partner_id, entity_type):
    return f"gdbm-{partner_id}/entity/{{{{ ds_nodash }}}}.*.{entity_type}.json"
```
WDYT?

**Comment by michalslowikowski00** [comment]
@potiuk I see... It it still WIP. I will fix right away.

File: airflow/providers/google/marketing_platform/example_dags/example_display_video.py (hunk 1 above)
Comment (michalslowikowski00) [reply airflow/providers/google/marketing_platform/example_dags/example_display_video.py:115]: Sure!

**Comment by michalslowikowski00** [comment]
Thanks @efolgar!

File: tests/providers/google/marketing_platform/operators/test_display_video_system.py [hunk 2]
```diff
@@ -26,14 +28,22 @@
...
+ENTITY_TYPE = "LineItem"
+
 
 @pytest.mark.system("google.marketing_platform")
 @pytest.mark.credential_file(GMP_KEY)
 class DisplayVideoSystemTest(GoogleSystemTest):
 
+    @staticmethod
+    def erf_bucket(partner_id, entity_type):
+        return f"gdbm-{partner_id}/entity/{{{{ ds_nodash }}}}.*.{entity_type}.json"
```
Comment (turbaszek) [inline tests/providers/google/marketing_platform/operators/test_display_video_system.py:41]: pylint will be sad due to no docstring, should we add type annotations as well?

File: airflow/providers/google/marketing_platform/example_dags/example_display_video.py (hunk 1 above)
Comment (turbaszek) [reply airflow/providers/google/marketing_platform/example_dags/example_display_video.py:115]: Once we 've got the method let's use it here :D

File: tests/providers/google/marketing_platform/operators/test_display_video_system.py (hunk 2 above)
Comment (michalslowikowski00) [reply tests/providers/google/marketing_platform/operators/test_display_video_system.py:41]: hurry is not a good approach... sorry. :(

Comment (michalslowikowski00) [reply tests/providers/google/marketing_platform/operators/test_display_video_system.py:41]: Yes, I will add docstring and types.

**Pull request closed without merging**